src/tcpip_checksum/jbuild

```diff
@@ -0,0 +1,30 @@
+(library
+ ((name        tcpip)
+  (public_name tcpip)
+  (modules (tcpip_checksum))
+  (libraries   (cstruct))
+  (wrapped false)))
+
+(library
+ ((name tcpip_xen)
+  (public_name tcpip.xen)
+  (libraries (tcpip))
+  (modules (tcpip_xen))
+  (c_names (checksum_stubs_xen))
+  (c_flags (:include c_flags_xen.sexp))
```

Comment:
ha yes https://github.com/janestreet/jbuilder/pull/153 fixes it for `flags` but it was already working for `c_flags`. Great :-)